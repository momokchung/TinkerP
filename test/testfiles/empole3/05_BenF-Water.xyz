    15 86_BenF-Water_1.00
     1  F      1.865439   -0.037079   -0.000227    94     2
     2  C      0.515308   -0.022013   -0.000105    95     1     3     4
     3  C     -0.137125    1.200148   -0.000160    96     8     2     5
     4  C     -0.164215   -1.229314    0.000063    96     2     9     6
     5  C     -1.530325    1.207005   -0.000041    97     3    10     7
     6  C     -1.557219   -1.205094    0.000180    97     4     7    11
     7  C     -2.241544    0.008737    0.000128    98     5     6    12
     8  H      0.438176    2.114297   -0.000286    99     3
     9  H      0.390597   -2.156031    0.000099    99     4
    10  H     -2.056614    2.151155   -0.000079   100     5
    11  H     -2.104439   -2.137269    0.000313   100     6
    12  H     -3.321948    0.020784    0.000221   101     7
    13  O      5.032976    0.039811    0.001448    16    14    15
    14  H      4.430145    0.021483    0.747475    17    13
    15  H      4.435596    0.023385   -0.748990    17    13
