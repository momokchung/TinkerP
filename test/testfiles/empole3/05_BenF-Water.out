
     ######################################################################
   ##########################################################################
  ###                                                                      ###
 ###            Tinker  ---  Software Tools for Molecular Design            ###
 ##                                                                          ##
 ##                      Version 8.10.5   February 2023                      ##
 ##                                                                          ##
 ##               Copyright (c)  Jay William Ponder  1990-2023               ##
 ###                           All Rights Reserved                          ###
  ###                                                                      ###
   ##########################################################################
     ######################################################################


 Intermolecular Energy :              -1.56109293 Kcal/mole

 Total Potential Energy :             -3.50704017 Kcal/mole

 Energy Component Breakdown :           Kcal/mole        Interactions

 Atomic Multipoles                    -3.50704017               72
